jspdf.js

```diff
@@ -177,6 +177,7 @@ var jsPDF = (function(global) {
    */
   function jsPDF(orientation, unit, format, compressPdf) {
     var options = {};
+    var vfs = getVfs();
```

Comment:
`var vfs = (typeof getVfs === "function") ? getVfs() : {}`

It is probably better if you add a function which adds the fonts by name and base64 instead of this separation breaking hack

Reply:
I just changed. Thanks